src/main/scala/strawman/collection/Seq.scala

```diff
@@ -18,20 +18,41 @@ trait LinearSeq[+A] extends Seq[A] with LinearSeqOps[A, LinearSeq, LinearSeq[A]]
 /** Base trait for indexed sequences that have efficient `apply` and `length` */
 trait IndexedSeq[+A] extends Seq[A] with IndexedSeqOps[A, IndexedSeq, IndexedSeq[A]]
 
+object IndexedSeq extends IterableFactory.Delegate[IndexedSeq](immutable.Vector)
+
 /** Base trait for Seq operations */
 trait SeqOps[+A, +CC[X], +C] extends Any
   with IterableOps[A, CC, C]
   with ArrayLike[A]
   with Equals {
 
+  /** Evidence that type `C` is (or can be converted to) an `IterableOnce[A]` */
+  implicit protected[this] def toCollection: C => IterableOnce[A]
```

Comment:
I fixed the implementation in the new PR (#115). Now the default implementation is based on the `reversed` protected method (which is also used by some `Traversable` methods, see #119). The `reversed` method builds a new collection by default. It is overriden in `IndexedSeq` to use an indexed view instead. There is no need for a `toCollection` method anymore.